tests/test_reservation.py

```diff
@@ -0,0 +1,12 @@
+from datetime import datetime
+from http import HTTPStatus
+
+
+def test_retrieve_status(client):
+    response = client.post("/api/reservations/", data={"location" : 1, "date" : datetime(1, 1, 1)})
+    assert response.status_code == HTTPStatus.OK
+
+
+def test_retrieve_returns_json(client):
```

Comment:
@carlosmiranda I believe you didn't understand my previous comment clearly :)
Let's do just one test in this way

```
def test_retrieve_reservation_list(client):
    response = client.post("/api/reservations/", data={"location" : 1, "date" : datetime(1, 1, 1)})
    assert response.status_code == HTTPStatus.OK
    assert response.is_json
```